src/Host.cpp

```diff
@@ -732,18 +721,7 @@ bool Host::closingDown()
     return shutdown;
 }
 
-<<<<<<< HEAD
-bool Host::installPackage(const QString& fileName, int module )
```

Comment:
It was the `orderShutDown()` function that was not needed, the above line was used but a space was added inside the opening parentheses '('.